src/chat/model_selector_loading.rs

```diff
@@ -0,0 +1,133 @@
+use makepad_widgets::*;
+
+live_design! {
+    import makepad_widgets::base::*;
+    import makepad_widgets::theme_desktop_dark::*;
+    import makepad_draw::shader::std::*;
+
+    import crate::shared::styles::*;
+    import crate::shared::widgets::*;
+    import crate::landing::model_card::ModelCard;
+
+    ANIMATION_SPEED = 1.5;
+
+    Bar = <RoundedView> {
```

Comment:
There is a bunch of repeated logic with other loading widget implemented, but decided to not overcomplicate this PR at this point.